### ISSUE TITLE
Terms of service

### ISSUE THREAD

**Issue by FriedrichFroebel** [sqlitebrowser/dbhub.io#239]
When signing up to the service, I am accepting the terms of service and privacy policy. I could not find both of them anywhere - the corresponding footers do not have any links.

Without any terms of service and privacy policy, it is rather intransparent what rights/permissions I am transferring, which types of applications are allowed or not (including API rate limits for example or making profit etc.) or what you are doing with the registration data and datasets I enter/upload/provide.

**Comment by justinclift** [comment]
Yeah, this has been on our ToDo list for ages.

Right wise, it's easily summarised to this (though we need to make it a proper document at some point):

* We don't sell, rent (or anything) people's data (eg personal info, interaction info, etc)
* We only require enough rights to uploaded data so we can run the DBHub.io and sqlitebrowser.org related services
  * For example, to copy data between our services, back stuff up, and to transmit the uploaded data in response to requests (website, api, db4s remote tab, etc)

"Term of Service" wise, it's pretty much common sense:

* This is a free service (for now), so we're doing it using our "best effort"
  * There might occasionally be downtime for things, though we're eliminating the causes for those are we come across them
* DO NOT do illegal stuff using our services
  * Probably best to avoid anything dodgy, as getting us in legal trouble isn't constructive at all. :wink:
* Don't abuse things.
  * There's no storage quota (for now), but if someone decides they're going to use us as a remote backup service for all their stuff (eg by stuffing their files into SQLite databases then uploading it every hour) then we'll nuke their account with no hesitation

If someone uses these services in some way to make a profit (legally!) then good on them.  That sounds like a win.  No issues with that at all. :smile:

That's the gist of things.  Is there any area which I've missed?

**Comment by FriedrichFroebel** [comment]
Thanks for the explanations. IANAL, but it does not feel right to be asked to accept some unwritten TOS/privacy policy during registration, so this probably should be improved in the short run (I stumbled upon this while looking at https://github.com/sqlitebrowser/sqlitebrowser/discussions/3537).

In your above explanations, it seems like the intended process for updating the policies is missing.

**Comment by justinclift** [comment]
> In your above explanations, it seems like the intended process for updating the policies is missing.

Good idea, hadn't thought of that.  The standard approach places seem to use is just updating the document with a "Last updated" date.

We'll have that think that through properly though.  Suggestions welcome of course. :smile:
